Libplanet.Explorer.Executable/Program.cs

```diff
@@ -135,6 +116,44 @@ public static async Task Main(string[] args)
             }
         }
 
+        private static IStore LoadStore(Options options)
+        {
+            bool readOnlyMode = options.Seeds is null;
+            IStore innerStore;
+            switch (options.StoreType)
+            {
+                case "rocksdb":
+                    innerStore = new RocksDBStore.RocksDBStore(options.StorePath);
+                    break;
+                default:
```

Comment:
I believe we should show error on this `default` case and have an explicit name (e.g., `"default"`) for `DefaultStore`.  In the current way, even if a user willing to use `RocksDBStore` passes any incorrect value (`rocks-db`, `RocksDB`, `rocksdbstore`, and so on) it won't show any error and will silently fall back to `DefaultStore`, which isn't not the user's intention.

Reply:
I tried to apply the suggestion in this commit, da2cc7ed350a0cec972e2686293f3a6d5928fde4, please check it was correct to your intention 🙏